File: SDKs/amazonaws.com/cloudfront/2017-03-25/python/src/sdk/models/shared/toomanycookienamesinwhitelist.py
import dataclasses
from typing import Optional


@dataclasses.dataclass
class TooManyCookieNamesInWhiteList:
    r"""TooManyCookieNamesInWhiteList
    Your request contains more cookie names in the whitelist than are allowed per cache behavior.
    """
    
    message: Optional[str] = dataclasses.field(default=None)
    

File: SDKs/amazonaws.com/cloudfront/2017-10-30/java/lib/src/main/java/openapisdk/models/shared/TooManyCookieNamesInWhiteList.java
package openapisdk.models.shared;



/**
 * TooManyCookieNamesInWhiteList
 * Your request contains more cookie names in the whitelist than are allowed per cache behavior.
**/
public class TooManyCookieNamesInWhiteList {
    public String message;
    public TooManyCookieNamesInWhiteList withMessage(String message) {
        this.message = message;
        return this;
    }
}